homestar-runtime/src/workflow/info.rs

```diff
@@ -330,10 +342,9 @@ impl Info {
     /// workflow [Cid].
     pub(crate) async fn gather<'a>(
```

Comment:
There is a comment issue above with `init`, as it's not calling `gather` anymore :).